### ISSUE TITLE
[DCI] change "action" column title wording to "request"

### ISSUE THREAD

**Issue by Clementinev** [blindnet-io/privacy-components-web#203]
On the DCI, the column title "action" refers to the technical wording, what do you think of changing it to "request" or "demand" ? 
<img width="1427" alt="Screenshot 2022-11-08 at 15 17 44" src="https://user-images.githubusercontent.com/89908145/200588369-ff9667bd-d7c5-4a5a-b834-a75b4c6d48c3.png">

**Comment by jboileau99** [comment]
@Clementinev I agree we shoud use something other than action, but I wonder if 'Demand' is too technical also. If we use 'Request', then it might be confusing because we use request at the top of the screen to refer to something else.

Maybe something like 'type'?

**Comment by Clementinev** [comment]
Yes "type" sounds good

**Comment by jboileau99** [comment]
Will be changed once #278 is merged :smile:
